src/Components/CreateEventPage.js

```diff
@@ -1,12 +1,217 @@
 import React, { Component } from 'react';
+import { MDBInput, MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+import './styles/CreateEventPage.css';
+import SubmitButton from './SubmitButton';
+import axios from "axios";
+import Select from 'react-select'
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import TimePicker from 'rc-time-picker';
+import ReactDOM from 'react-dom';
+import 'rc-time-picker/assets/index.css';
 
+const options = [
```

Comment:
Add a TODO comment to integrate exhaustive list if you are not planning on implementing it.

Reply:
Because we'll probably want some "commons" or "config" folder to store stuff like the location list etc. Makes more sense than hardcoding it in 100 different places.